picoCTF-shell/hacksport/deploy.py

```diff
@@ -70,7 +75,8 @@ def give_port():
             "All usable ports are taken. Cannot deploy any more instances.")
 
     while True:
```

Comment:
If banned ports and used_ports are merged, also don't need to loop `while True` anymore since random.choice should ensure no collisions.